Domain.Sql/StorableEvent.cs

```diff
@@ -21,12 +21,13 @@ public StorableEvent()
             Timestamp = DateTimeOffset.UtcNow;
         }
 
+        [Index(IsUnique = true), Index("IX_Id_StreamName_Type", 1, IsUnique = true)]
         public long Id { get; set; }
 
-        [MaxLength(255)]
+        [MaxLength(50), Index("IX_Id_StreamName_Type", 2, IsUnique = true)]
```

Comment:
How did you decide on these lengths?

Why did you specify the index name, instead of taking the (reasonable) EF default?